aips/aip-4.md

```diff
@@ -0,0 +1,78 @@
+# AIP-4: Multi-Composable Token Standard
```

Comment:
Thank you for submitting the PR. please also create a discussion issue like this one https://github.com/aptos-foundation/AIPs/issues/15. It is easier to keep the discussion in an issue

Reply:
Hey @JacobADevore could you please add the header like other AIPs? Let's make this one AIP-5. once you updated, we can land this draft and iterate on it

Reply:
> Hey @JacobADevore could you please add the header like other AIPs? Let's make this one AIP-5. once you updated, we can land this draft and iterate on it

Sure thing! Does that suffice?

Reply:
You need to update the file name too

Reply:
Got it! Sorry about that.